pkg/pubsub/topic_handler.go

```diff
@@ -0,0 +1,37 @@
+package pubsub
```

Comment:
@restevens402 a generic boilerplate topic handler to be used with topics created and subscribed to on the fly.